### ISSUE TITLE
Pull request stuck on CLA bot

### ISSUE THREAD

**Issue by geofft** [onnx/onnx#4132]
Pull request #3667 has been stuck on the CLA bot for eight months.

As noted there, https://github.com/onnx/onnx/blob/main/CONTRIBUTING.md says that ONNX uses a DCO and does not require a CLA. Should the CLA bot be running at all?

**Comment by jcwchen** [comment]
Hi @geofft,
Thank you for pointing out. According to the document, yes CLA should be removed in ONNX. I will follow-up it with @prasanthpul and meanwhile will review your PR as well.

**Comment by jcwchen** [comment]
Update: actually it has been removed a year ago, but it wasn't removed entirely due to some CLA [issues](https://github.com/cla-assistant/cla-assistant/issues/691). I think at least after that moment CLA is not required anymore, but the robot sometimes kept notifying contributors to sign-off CLA, which was quite confusing. 

Therefore we took further actions to try to remove it entirely according to the instruction [here](https://github.com/solvespace/solvespace/issues/1098#issuecomment-1008696674). I hope this time we have removed it entirely. At least, this fresh merged [PR](https://github.com/onnx/onnx/pull/4157), which is contributed by a first-time contributor, didn't get any CLA notification. Still, just in case, I will wait for a while and keep this issue open to monitor whether CLA Assistant disappear for good in this repo. Thank you again for catching this.
